recipes/click-didyoumean/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "click-didyoumean" %}
+{% set version = "0.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 112229485c9704ff51362fe34b2d4f0b12fc71cc20f6d2b3afabed4b8bfa6aeb
+
+build:
+  noarch: python
+  number: 0
+  script: '{{ PYTHON }} -m pip install . -vv'
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools
```

Comment:
You should remove setuptools, since it is a pip dep

Reply:
Since `setuptools` is directly referenced in the `setup.py` file for this package (and the others) I would rather not rely upon the fact that `pip` happens to rely upon `setuptools` right now.

I can imagine a glorious future when `pip` is build-library agnostic...

Reply:
Having setuptools and pip as deps can interfere and cause problems.  It is general policy in conda-forge to not have both of them

Reply:
I have tens of recipes where I have manually specified `setuptools` like this, and have had them approved here. However, if this is a new policy, I am happy to follow it.